packages/tracing/src/transaction.ts

```diff
@@ -98,14 +103,15 @@ export class Transaction extends SpanClass implements TransactionInterface {
       }).endTimestamp;
     }
 
-    const transaction: Event = {
+    const transactionEvent: Event = {
       contexts: {
         trace: this.getTraceContext(),
       },
       spans: finishedSpans,
       start_timestamp: this.startTimestamp,
       tags: this.tags,
       timestamp: this.endTimestamp,
+      tracestate: this.tracestate,
```

Comment:
Actually, after talking with Daniel about this same issue in another PR (since I wrote the above), we agreed on passing tags called `__sentry_x` and then filtering all such tags out before the event is sent. I was thinking I'd adjust that for this usecase later, but I suppose I might as well do it here. Will leave this unresolved until I push that fix.